Assets/Mirage/Runtime/NetworkIdentity.cs

```diff
@@ -134,7 +134,7 @@ public sealed class NetworkIdentity : MonoBehaviour
         /// This returns true if this object is the one that represents the player on the local machine.
         /// <para>This is set when the server has spawned an object for this particular client.</para>
         /// </summary>
-        public bool IsLocalPlayer => ClientObjectManager != null && ClientObjectManager.LocalPlayer == this;
+        public bool IsLocalPlayer => Client != null && Client.Player.Identity == this;
```

Comment:
```suggestion
        public bool IsLocalPlayer => Client != null && Client.Player?.Identity == this;
```

I think this one needs to be nullable as well

Reply:
good catch